picard/ui/__init__.py

```diff
@@ -30,6 +32,36 @@ def __init__(self, parent=None):
         super().__init__(parent, self.flags)
 
 
+class SaveGeometry(object):
```

Comment:
This should be named as some noun, given the default class naming convention. SaveGeometry is more of a function/decorator name. How about ResizableDialog or something along those lines since it is applied only to dialogs.

Reply:
Infact, I dont find a single PicardDialog (You missed a couple in ui/passworddialog that can also be included to inherit from this) that doesn't implement this functionality. Seems like we can merge this code with PicardDialog and inherit from a single class.

Reply:
Fixed in c0e27f0ee19d389cbcc5873a2b1a2d1fadc083f3